pyasic/miners/backends/auradine.py

```diff
@@ -386,3 +386,27 @@ async def _get_uptime(self, rpc_summary: dict = None) -> Optional[int]:
                 return rpc_summary["SUMMARY"][0]["Elapsed"]
             except LookupError:
                 pass
+
+    async def upgrade_firmware(self, url: str = None, version: str = "latest") -> dict:
```

Comment:
This is good, but ideally the call API should be consistent across type (usually I do this with a method in the `BaseMiner` that keeps the way it is used consistent).  The complexity here is that this miner doesn't really take local files, so compared to the other miners it might be a bit weird.  

See if you can come up with a good way to make the call API consistent across miner types for this.